src/test/java/nextstep/subway/unit/LineTest.java

```diff
@@ -52,4 +58,41 @@ void removeSection() {
 		//then
 		assertThat(line.getSections()).isEmpty();
 	}
+
+	@DisplayName("마지막 구간 정보 조회 테스트")
+	@Test
+	void getLastSectionTest() {
+
+	    //given
+		Line line = new Line("분당선", "yellow");
+		Section section = new Section(line, new Station("서현역"), new Station("이매역"), 10);
+		line.addSection(section);
+
+	    //when
+		Section lastSection = line.getLastSection();
+
+	    //then
+		assertThat(lastSection.getUpStation().getName()).isEqualTo("서현역");
+		assertThat(lastSection.getDownStation().getName()).isEqualTo("이매역");
```

Comment:
```suggestion
		assertThat(lastSection.getUpStation().getName()).isEqualTo(서현역.getName());
		assertThat(lastSection.getDownStation().getName()).isEqualTo(이매역.getName());
```

이런식으로 작성하는게 어떨까요 매번 하드코딩으로 검증을하면 휴먼에러 문제에서 벗어나기 힘들거든요.